test/unit/services/workers/test-send-advance-claim-evidence-reminder-notification.js

```diff
@@ -9,15 +9,20 @@ const EMAIL_ADDRESS = 'test@test.com'
 const REFERENCE = '1234567'
 const ELIGIBILITY_ID = 4321
 const CLAIM_ID = 1234
+var firstName = 'Joe'
```

Comment:
The firstName parameter should be a constant because it's contents will never change.  Change this to `const FIRST_NAME = 'Joe'`